src/core/less/variables/devices.less

```diff
@@ -0,0 +1,15 @@
+// THIS IS WHERE WE CAN DECLARE ANY OTHER VARIABLES (ie. device-width, border-width etc.)
+
+
+// Device widths
+// =============
+
+@device-width-small: 520px;
```

Comment:
change value to @adapt-device-small etc see. https://github.com/adaptlearning/adapt_framework/blob/master/grunt/tasks/less.js#L28-L30